### PR TITLE
Updates to empty namespace docs

### PR DESCRIPTION
**Pull request by stefanrbk**
## Summary

Added summaries to the following:
`System.Runtime.Intrinsics`
`System.Runtime.Intrinsics.X86`
`System.Runtime.Intrinsics.Arm`

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=5055) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=5055) before we can accept your contribution.<br/><br/>:x: stefanrbk [sign now](https://cla.dotnetfoundation.org/dotnet/dotnet-api-docs?pullRequest=5055)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/dotnet-api-docs?pullRequest=5055) it.</sub>

File: xml/ns-System.Runtime.Intrinsics.Arm.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.Arm">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains classes representing the different SIMD instruction sets available to the ARM architecture. Not all of these instructions sets are available on all ARM CPUs though. Check the "IsSupported" property on each class to determine compatibility with the executing CPU.</summary>
```
Comment (tannergooding) [inline xml/ns-System.Runtime.Intrinsics.Arm.xml:3]: This, and the same for x86, could likely be reworded slightly. The underlying APIs largely touch SIMD code, but there are also other APIs which aren't SIMD at all.

Perhaps this could say something like:
> Exposes select instructions available on ARM (branding needed?) hardware. These instructions are divided into separate classes representing the underlying ISAs (Instruction Set Architectures) exposed by the platform. Support for an ISA can be queried via the `IsSupported` property exposed on the respective type and may be false if the underlying hardware does not support it or if it is not currently supported by the underlying runtime.

I

Comment (tannergooding) [reply xml/ns-System.Runtime.Intrinsics.Arm.xml:3]: Some of this feels more like conceptual docs however and it is a bit more wordy than most summaries.

File: xml/ns-System.Runtime.Intrinsics.X86.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.X86">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains classes representing the different SIMD instruction sets available to the x86 architecture. Not all of these instructions sets are available on all x86 CPUs though. Check the "IsSupported" property on each class to determine compatibility with the executing CPU.</summary>
```
Comment (tannergooding) [inline xml/ns-System.Runtime.Intrinsics.X86.xml:3]: Not sure if it's worth clarifying that x86 includes x64 in this context.

Comment (stefanrbk) [reply xml/ns-System.Runtime.Intrinsics.X86.xml:3]: I added x64 in my latest commit.

File: xml/ns-System.Runtime.Intrinsics.Arm.xml (hunk 1 above)
Comment (stefanrbk) [reply xml/ns-System.Runtime.Intrinsics.Arm.xml:3]: I agree. I reworded it, mixing your phrasing with mine. The revised phrasing is able to drop the comment about `IsSupported` if you deem it too wordy still.

File: xml/ns-System.Runtime.Intrinsics.Arm.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.Arm">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Exposes select instruction set extensions available for ARM systems. These instruction sets are expressed as separate classes for each extension. Support for any extension within the current environment can be determined by querying the "IsSupported" property on the respective type.</summary>
```
Comment (gewarren) [inline xml/ns-System.Runtime.Intrinsics.Arm.xml:3]:
```suggestion
    <summary>Exposes select instruction-set extensions for ARM systems. These instruction sets are expressed as separate classes for each extension. Support for any extension within the current environment can be determined by querying the <see langword="IsSupported" /> property on the respective type.</summary>
```

File: xml/ns-System.Runtime.Intrinsics.X86.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Runtime.Intrinsics.X86">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Exposes select instruction set extensions available for x86 and x64 systems. These instruction sets are expressed as separate classes for each extension. Support for any extension within the current environment can be determined by querying the "IsSupported" property on the respective type.</summary>
```
Comment (gewarren) [inline xml/ns-System.Runtime.Intrinsics.X86.xml:3]:
```suggestion
    <summary>Exposes select instruction-set extensions for x86 and x64 systems. These instruction sets are expressed as separate classes for each extension. Support for any extension within the current environment can be determined by querying the <see langword="IsSupported" /> property on the respective type.</summary>
```

File: xml/ns-System.Runtime.Intrinsics.xml [hunk 5]
```diff
@@ -1,6 +1,11 @@
 <Namespace Name="System.Runtime.Intrinsics">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types used to create and convey register states in various sizes and formats for use with instruction set extensions.
```
Comment (gewarren) [inline xml/ns-System.Runtime.Intrinsics.xml:3]:
```suggestion
    <summary>Contains types used to create and convey register states in various sizes and formats for use with instruction-set extensions.
```

**Review by gewarren — APPROVED** [review]

**Review by gewarren — DISMISSED** [review]

File: xml/ns-System.Runtime.Intrinsics.xml [hunk 6]
```diff
@@ -1,6 +1,11 @@
 <Namespace Name="System.Runtime.Intrinsics">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Contains types used to create and convey register states in various sizes and formats for use with instruction-set extensions.
+    
+    For the instructions to manipulate these registers, see <see cref="N:System.Runtime.Intrinsics.X86" /> and <see cref="N:System.Runtime.Intrinsics.Arm" />.
```
Comment (gewarren) [inline xml/ns-System.Runtime.Intrinsics.xml:5]:
```suggestion
For the instructions to manipulate these registers, see <see cref="N:System.Runtime.Intrinsics.X86" /> and <see cref="N:System.Runtime.Intrinsics.Arm" />.
```

**Review by gewarren — APPROVED** [review]
Preview looks good now.
